wiki/en/Running-a-Server.md

```diff
@@ -125,185 +144,174 @@ To register with one of the Directories built into the Jamulus Client, replace `
 
-To send these signals using systemd, create the following two `.service` files in `/etc/systemd/system`, calling them something appropriate (e.g. `jamulusTogglerec.service`).
+##### `--directoryfile` 
+_Directories only:_ Remember registered Servers even if the Directory is restarted. See [the Directories guide](Directories) for further information.
 
-**Note:** You will need to save recordings to a path _outside_ of the jamulus home Directory, or remove `ProtectHome=true` from your systemd unit file (be aware that doing so is however a potential security risk).
+##### `-f or --listfilter` 
+_Directories only:_ Whitelist Servers registering on the Server. See [the Directories guide](Directories) for further information.
 
-For turning recording on or off (depending on the current state):
-
-~~~
- [Unit]
- Description=Toggle recording state of Jamulus Server
- Requisite=Jamulus-Server
+---
 
- [Service]
- Type=oneshot
- ExecStart=/bin/systemctl kill -s SIGUSR2 Jamulus-Server
-~~~
+### General Server options
+These options can be used regardless of which mode your server is running in (although some may not be relevant for Directories).
 
-For starting a new recording:
+##### `-d or --discononquit` 
+Normally, when a Server is stopped or restarted, any Clients that have not used their “Disconnect” buttons will re-establish connection when the Server comes back up again. Using this option forces Clients to manually re-establish their connections to the Server.
 
-~~~
- [Unit]
- Description=Start a new recording on Jamulus Server
- Requisite=Jamulus-Server
+##### `-F or --fastupdate` 
+Reduces latency if Clients connect with “Enable Small Network Buffers” option. Requires faster CPU to avoid dropouts, and more bandwidth to enabled Clients.
 
- [Service]
- Type=oneshot
- ExecStart=/bin/systemctl kill -s SIGUSR1 Jamulus-Server
-~~~
+##### `-l or --log` 
+Enable logging, set path and file name
 
-_Note: The Jamulus service name in the `ExecStart` line needs to be the same as the `.service` file name you created when setting systemd to control your Jamulus Server. So in this example it would be `Jamulus-Server.service`_
+##### `-m or --htmlstatus` 
+Enable HTML status file, set path and file name
 
-Run `sudo systemctl daemon-reload` to register them for first use.
+##### `-P or --delaypan` 
+Start with delay panning enabled.This option uses small differences in sound arrival time between the two ears. It produces a stereo effect similar to natural human hearing when compared to normal “volume” panning.
 
-Now you can run these with the `systemctl` command, for example:
+##### `-s` or `--server`
+Start Jamulus in Server mode
 
-`sudo systemctl start jamulusTogglerec` (assuming you named your unit file `jamulusTogglerec.service`)
+##### `--serverbindip` 
+Specify the IP address the Jamulus process will bind to
```

Comment:
This needs explaining or a link to an explanation.

Reply:
_Chosen_, or _specified_.

Reply:
Why is "all addresses" in inverted commas though?